src/components/Products.js

```diff
@@ -0,0 +1,34 @@
+// PRODUCTS COMPONENT //
+
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import { Navigation } from 'swiper/modules';
+
+export default function Products({itemsForDisplay}) {
+  
+  const cards = itemsForDisplay.map(item => {
+    return (
+        <SwiperSlide key={item.name}>
+          <div className="card">
+            <img src={item.image} alt={item.name} className="card__image" />
```

Comment:
You may already know this, but wanted to point out that the item image for dog toy is broken. I console.logged it and it is the only one that is in a .jpg format, rather than being a url.

Reply:
good catch